storybook/storybook-react/src/components/grid/PageGrid.stories.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { GridCell, PageGrid, Paragraph } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+
+import '@amsterdam/design-system-css/src/grid/page-grid.scss'
+import '@amsterdam/design-system-css/src/grid/grid-cell.scss'
+import '@amsterdam/design-system-css/src/heading/heading.scss'
+import '@amsterdam/design-system-css/src/paragraph/paragraph.scss'
+import '../../../../storybook-css/src/docs/stories.scss'
+
+const meta = {
+  title: 'PageGrid',
+  id: 'page-grid',
+  component: PageGrid,
+} satisfies Meta<typeof PageGrid>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    children: (
+      <>
+        {Array.from(Array(12).keys()).map((i) => (
+          <GridCell key={i}>
+            <div className="amsterdam-docs-pink-box">
```

Comment:
I think we can add the `amsterdam-docs-pink-box` class directly on the `Paragraph` component, no? Cleans up the code view in SB a bit

Reply:
The class name sets a padding, so it’s more suitable for a container than for a paragraph. We’ll improve on this concept in the near future.